pkgs/tools/admin/oci-cli/default.nix

```diff
@@ -73,6 +76,25 @@ buildPythonApplication rec {
       --replace "oci==2.78.0" "oci"
   '';
 
+  postInstall = ''
+    # Similar to the gcloud CLI
+    cat >oci.zsh <<EOF
+    #compdef oci
```

Comment:
Isn't this line unused?

Reply:
This is a special tag required for auto-loading

https://zsh.sourceforge.io/Doc/Release/Completion-System.html#index-completion-system_002c-autoloaded-functions